crates/astria-core/src/oracles/price_feed/market_map.rs

```diff
@@ -1,10 +1,7 @@
 pub mod v2 {
-    use std::{
-        io::{
-            self,
-            Write,
-        },
-        str::FromStr,
+    use std::io::{
+        self,
+        Write,
```

Comment:
Just a stylistic comment: I'd import `io` and then just refer to it as `io::Write` (I don't see the rest of the file, but I assume that somewhere there is an `impl Write for *`

Reply:
Three more LoC deleted - nice! :)

Done in [7d16f72](https://github.com/astriaorg/astria/pull/2113/commits/7d16f72232eacf4103249ea655569663e419397e).